tests/test_main.py

```diff
@@ -1,6 +1,11 @@
+import mock
+import sys
+
 from validator.main import main
 
-def test_main():
+
+@mock.patch.object(sys, 'stderr')
```

Comment:
Can you please explain what issues you had? I'd rather have a fix for the cause, not the consequences. I'm afraid this will trip us in the future, wondering why we don't get the output we're expecting?